nixos/modules/tasks/auto-upgrade.nix

```diff
@@ -63,6 +63,17 @@ let cfg = config.system.autoUpgrade; in
         '';
       };
 
+      randomizedDelaySec = mkOption {
+        default = "0";
+        type = types.str;
+        example = "45min";
+        description = ''
+          Add a randomized delay to wait before automatically upgrading.
+          The delay will be between zero and this value.
+          This value must be a valid systemd timespan.
```

Comment:
Could you reference the systemd.time manual as done in the `dates` option description?